src/ds/address.h

```diff
@@ -70,7 +70,7 @@ namespace snmalloc
    * power of two.
    */
   template<size_t alignment, typename T = void>
-  SNMALLOC_FAST_PATH T* pointer_align_down(void* p)
+  SNMALLOC_FAST_PATH T* pointer_align_down(const void* p)
```

Comment:
Notice that this result in an inconsistent qualifier with other alignment cast operations.